### DISCUSSION TITLE
https://penguins-eggs.net/blog/lastoslinux

### DISCUSSION THREAD

**Discussion by giscus** [pieroproietti/penguins-blog#19] [General]
# https://penguins-eggs.net/blog/lastoslinux

LastOSLinux - The methods behind the madness (Custom Distribution Download)

https://penguins-eggs.net/blog/lastoslinux

**Comment by pieroproietti** [comment]
@LiveFreeDead

Fine Glenn, I see your passion inside, a great piece!
